### PR TITLE
document new Activity.SetStatus API

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #
* [ ] Changes in public API reviewed

### CODE REVIEW

File: src/OpenTelemetry.Api/README.md [hunk 1]
```diff
@@ -392,8 +392,26 @@ corresponding overloads of `ActivityEvent`.
...
+activity?.SetStatus(ActivityStatusCode.Error, "Error Description");
+```
+
+All the official
+[Exporters](../../docs/trace/extending-the-sdk/README.md#exporter) shipped from
+this repo supports the above shown mechanism of setting status.
+
+#### Setting Status - DiagnosticSource version older than 6.0
+
+Prior to [DiagnosticSource
```
Comment (reyang) [inline src/OpenTelemetry.Api/README.md:411]: Do we even need this? (e.g. the 1.2.0 version of the SDK would require S.D.DS >= 6.0?)

File: src/OpenTelemetry.Api/README.md [hunk 2]
```diff
@@ -392,8 +392,26 @@ corresponding overloads of `ActivityEvent`.
 
 OpenTelemetry defines a concept called
 [Status](https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/trace/api.md#set-status)
-to be associated with `Activity`. There is no `Status` class in .NET, and hence
-`Status` is set to an `Activity` using the following special tags:
+to be associated with `Activity`. Starting with [DiagnosticSource
+6.0](https://www.nuget.org/packages/System.Diagnostics.DiagnosticSource/6.0.0),
+`SetStatus` API on `Activity` may be used to set status and status description
```
Comment (reyang) [inline src/OpenTelemetry.Api/README.md:397]:
```suggestion
`SetStatus` API on `Activity` can be used to set the status and description
```

**Review by reyang — APPROVED** [review]

File: src/OpenTelemetry.Api/README.md [hunk 3]
```diff
@@ -392,8 +392,26 @@ corresponding overloads of `ActivityEvent`.
...
+as shown below:
+
+```csharp
+activity?.SetStatus(ActivityStatusCode.Ok);
+activity?.SetStatus(ActivityStatusCode.Error, "Error Description");
+```
+
+All the official
+[Exporters](../../docs/trace/extending-the-sdk/README.md#exporter) shipped from
+this repo supports the above shown mechanism of setting status.
```
Comment (utpilla) [inline src/OpenTelemetry.Api/README.md:407]: nit: support*

**Review by utpilla — APPROVED** [review]

File: src/OpenTelemetry.Api/README.md (hunk 1 above)
Comment (cijothomas) [reply src/OpenTelemetry.Api/README.md:411]: leaving it here until 1.2 stable is out.

**Pull request merged**